src/bridge/bridgemain.h

```diff
@@ -765,6 +771,7 @@ typedef struct
     X87STATUSWORDFIELDS x87StatusWordFields;
     X87CONTROLWORDFIELDS x87ControlWordFields;
     LASTERROR lastError;
+    LASTSTATUS lastStatus;

```

Comment:
The main issue with this is that any plugins using this structure will break so I'd rather not do that...

A solution for this is to let the plugin pass the buffer size so we can make sure no buffers are being overwritten but it's a bit too late for that now 😢

Reply:
You can introduce `GetFoobarEx` like WinAPI does, when `GetFoobar` will call `GetFoobarEx(buff, original_size)`

Reply:
Yep, that's what I was afraid of. If only `LASTERROR` had 4 more bytes of padding before `name` :(

What do you suggest to do about this? It is of course possible to add a new bridge export only to retrieve the last status value, but I think we can all agree that would be pretty horrible :P Possibly worse: leave the public header unchanged but secretly give the register view the bigger struct anyway because it knows about the status value. Ergh.

@justanotheranonymoususer's approach sounds like the only reasonable long term solution to me. E.g. `BRIDGE_IMPEXP bool DbgGetRegDumpEx(REGDUMP* regdump, size_t bufferSize);` where `bufferSize` is `sizeof(REGDUMP)`, but this value would of course depend on the plugin SDK version used.

I can also see other reasons for doing this besides adding the last status value: if AVX512 support is added in the future, `REGISTERCONTEXT` will need to be expanded with the zmm registers, which also affects the size of `REGDUMP`.

Reply:
I think `DbgGetRegDumpEx` would be the better solution. Internally we can change `_dbg_getregdump` to take an additional `size_t bufferSize` like you suggested to keep the change as small as possible...

Reply:
Hmm, so you would suggest keeping one `_dbg_getregdump` internally (with an added `bufferSize` parameter) that is used by both `DbgGetRegDump` and `DbgGetRegDumpEx`? Just making sure as it's probably easier if I submit that change as a separate PR.